tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaAuthorizationTestBase.java

```diff
@@ -202,7 +202,8 @@ public void testAuthorizationSuccess() throws PulsarAdminException {
 
```

Comment:
Are the changes of this file and `KafkaNonPartitionedTopicTest.java` related to this PR?

Reply:
The previous change introduced a new system topic, when we do the `listTopics`, the system topic is in the result, is should add in the next PR. We can remove it for now.